### PR TITLE
Changes were made to the HTML h tags - Issue 1451

### PR DESCRIPTION
**Pull request by martaribeiro**
Fixes # 1451.

Changes proposed in this PR:
- Changes were made to the HTML h tags in order to make it more accessibility friendly and compliant

### CODE REVIEW

File: app/views/answers/_locking.html.erb [hunk 1]
```diff
@@ -1,6 +1,9 @@
+<!--
+  This partial displays when a question was edit by two or more users. The local variables are: answer, question, readonly, locking
```
Comment (briri) [inline app/views/answers/_locking.html.erb]: thanks for adding these comments!

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
               <%= _("Do you have a %{application_name} account?") % {
                 application_name: Rails.configuration.branding[:application][:name]} %>
-            </h3>
-            <p class="text-center">
+            </p>
+            <p class="text-center fontsize-h3 color-heading-text">
               <i class="fa fa-arrow-circle-down" aria-hidden="true"></i>
             </p>
             <h2 class="text-center">
-                <%= _("Sign in") %></i>
```
Comment (briri) [inline app/views/devise/registrations/new.html.erb]: good catch

File: app/views/phases/_overview.html.erb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <%# locals: { phase } %>
 <div class="row" id="<%= dom_id(phase) %>">
   <div class="col-xs-12">
-    <h4>
+    <h4 class=''>
```
Comment (briri) [inline app/views/phases/_overview.html.erb]: don't need the class here if its empty

**Comment by martaribeiro** [comment]
I have fixe the empty class. Which should now be a <p> tag with a class and not an <h> tag

**Pull request merged**